website/src/docs/uppy.md

```diff
@@ -361,8 +361,10 @@ data = {
 ```
 
 ```javascript
-uppy.on('upload-progress', (data) => {
```

Comment:
the `data` object snippet still has an `id` up here—i think we can probably remove it entirely because you added the object shape in a comment in this code block.